classes/class-wc-connect-taxjar-integration.php

```diff
@@ -204,22 +204,14 @@ public function configure_tax_settings() {
 	}
 
 	/**
-	 * TaxJar supports USA, Canada, Australia, and the European Union
+	 * TaxJar supports USA, Canada, Australia, and the European Union + Great Britain
 	 * See: https://developers.taxjar.com/api/reference/#countries
 	 *
 	 * @return array Countries supported by TaxJar.
 	 */
 	public function get_supported_countries() {
-		$supported_countries = array_merge(
-			array(
-				'US',
-				'CA',
-				'AU',
-			),
-			WC()->countries->get_european_union_countries()
```

Comment:
There is a `apply_filters( 'woocommerce_european_union_countries', $countries, $deprecated );` in core. I don't think it breaks anything if we no longer relies on it. Checking.

Reply:
I think this filter is here for 3rd party compatibility. The TaxJar implementation we have is kinda separated from this list. 
One thing I can think of is if this filter return a subset (ie. just 'US') instead of the full list, and WCS continue to return all taxes. 

Even then, I think that's fine.